skyportal/handlers/api/analysis.py

```diff
@@ -1159,6 +1206,15 @@ def get(self, analysis_resource_type, analysis_id=None):
             True,
             1,
         ]
+
+        summary_only = self.get_query_argument("summaryOnly", False) in [
+            "True",
+            "t",
+            "true",
+            "1",
+            True,
+            1,
+        ]
```

Comment:
The baselayer `get_query_argument` handles these booleans internally now, so if the default is boolean then the output is boolean, too.

Reply:
fixed.